feat(tracing): Add `any_agent/tracing/attributes`. (#614)
mozilla-ai/any-agent: docs/agents/callbacks.md

Advanced designs such as safety guardrails or custom side-effects can be integrated into your agentic system using this functionality.

## Context

During the agent run ( [`agent.run_async`][any_agent.AnyAgent.run_async] or [`agent.run`][any_agent.AnyAgent.run] ), a unique [`Context`][any_agent.callbacks.context.Context] object is created and shared across all callbacks.

`any-agent` populates the [`Context.current_span`][any_agent.callbacks.context.Context.current_span]
property so that callbacks can access information in a framework-agnostic way.

> You can check the attributes available for LLM Calls and Tool Executions in the [`GenAI`][any_agent.tracing.attributes.GenAI].

Reviewer: Something like
```suggestion
You can see what attributes are available for LLM Calls and Tool Executions by examining the [`GenAI`][any_agent.tracing.attributes.GenAI] class.
```
